en/docs/reference/mediators/property-reference/http-transport-properties.md

```diff
@@ -548,10 +548,10 @@ HTTP response's Reason- Phrase that is sent by the backend. For example, if the
 <td><p><strong>Description</strong></p></td>
 <td><div class="content-wrapper">
 <p>If the request sent by the client contains the ‘Content-Length’ header, this property allows the Micro Integrator to send the request with the content length (without HTTP chunking) to the back end server.</p>
-<p>You should set this to true in scenarios where the backend server is not able to accept chunked content. For example, in a scenario where a pass-through proxy is defined and the backend does not accept chunked content, this property should be used together with the <a href="../../../../references/mediators/property-reference/http-transport-properties/#copy_content_length_from_incoming">COPY_CONTENT_LENGTH_FROM_INCOMING</a> property, to simply add the content length without chunking.</p>
+<p>You should set this to true in scenarios where the backend server is not able to accept chunked content. For example, in a scenario where a pass-through proxy is defined and the backend does not accept chunked content, this property should be used together with the <a href="{{base_path}}/references/mediators/property-reference/http-transport-properties/#copy_content_length_from_incoming">COPY_CONTENT_LENGTH_FROM_INCOMING</a> property, to simply add the content length without chunking.</p>
 <p>When HTTP 1.1 is used, this property disables chunking and sends the content length. When HTTP 1.0 is used, the property only sends the content length.</p>
 <b>Note</b>:
-<p>This property can cause performance degradation, and thereby, you should only use it with message relay. If you set this to true, the Micro Integrator forwards the content length coming from the client request to the backend without building the message and calculating the content length. Since the message doesn’t get build, using these properties will perform better than using <a href="../../../../references/mediators/property-reference/http-transport-properties/#disable_chunking">DISABLE_CHUNKING</a> . However, if you change the receiving payload before sending it to the backend, then having this property will result in an error due to a content length mismatch.</p>
+<p>This property can cause performance degradation, and thereby, you should only use it with message relay. If you set this to true, the Micro Integrator forwards the content length coming from the client request to the backend without building the message and calculating the content length. Since the message doesn’t get build, using these properties will perform better than using <a href="{{base_path}}/references/mediators/property-reference/http-transport-properties/#disable_chunking">DISABLE_CHUNKING</a> . However, if you change the receiving payload before sending it to the backend, then having this property will result in an error due to a content length mismatch.</p>
```

Comment:
```suggestion
<p>This property can cause performance degradation, and thereby, you should only use it with message relay. If you set this to true, the Micro Integrator forwards the content length coming from the client request to the backend without building the message and calculating the content length. Since the message doesn’t get build, using these properties will perform better than using <a href="{{base_path}}/reference/mediators/property-reference/http-transport-properties/#disable_chunking">DISABLE_CHUNKING</a> . However, if you change the receiving payload before sending it to the backend, then having this property will result in an error due to a content length mismatch.</p>
```